doc/dynamo0p3.rst

```diff
@@ -182,8 +190,11 @@ in the :ref:`kernel-layer` section. This section explains the
 dynamo0.3-specific rules for kernels and then goes on to describe
 their metadata and subroutine arguments.
 
-Rules
-+++++
+General Rules
++++++++++++++
+
+In the following, 'operator' refers to both LMA and CMA operator
```

Comment:
Is LMA the same as a traditional operator?  The above line makes it sound like there are only LMA and CMA operators. If so, we need to use the names consistently in the documentation i.e. it should be referred to either as an operator or as LMA. If not, then operator refer to a "standard" operator (should this have a TLA too?) as well as LMA and CMA.